### DISCUSSION TITLE
Does this project have any design with prototype?

### DISCUSSION THREAD

**Discussion by imranertaza** [edrisranjbar/qurandl#29] [Ideas]
Hi
It is better for all the contributors if there is any design of the project along with the prototype. Example, if you have any figma/xd/photoshop design of the project, could you please share the design?
Thank you

**Comment by edrisranjbar** [comment]
we do have a sketch for home page, which is going to be a single web application which has main parts for version alpha. this is the Figma url: [FIGMA](https://www.figma.com/file/4ALz53VRSLFrkdXTSy1ywf/qurandl?type=design&node-id=0%3A1&mode=design&t=mkVV65NmQ2khEgkA-1)

**Comment by imranertaza** [comment]
It's just the sketch of one page. Just the idea. It would better if we have any complete design like this below.
https://www.figma.com/file/roliceKAxoePkfe7p0YqTR/Amazing-gadgets?type=design&node-id=0%3A1&t=us92penjcYojHgC5-1
